Corona-Warn-App/src/main/java/de/rki/coronawarnapp/main/CWASettings.kt

```diff
@@ -24,11 +25,18 @@ class CWASettings @Inject constructor(
         get() = prefs.getBoolean(PKEY_DEVICE_TIME_INCORRECT_ACK, false)
         set(value) = prefs.edit { putBoolean(PKEY_DEVICE_TIME_INCORRECT_ACK, value) }
 
+    val lastAppVersion = prefs.createFlowPreference(
+        key = LAST_APP_VERSION,
+        defaultValue = DEFAULT_APP_VERSION
+    )
+
     fun clear() {
         prefs.clearAndNotify()
     }
 
     companion object {
         private const val PKEY_DEVICE_TIME_INCORRECT_ACK = "devicetime.incorrect.acknowledged"
+        private const val LAST_APP_VERSION = "last.installed.version"
+        private const val DEFAULT_APP_VERSION = 1.0.toLong()
```

Comment:
This `1.0.toLong()` is confusing to me. `2.0` is a `double` , but then we cast it to `long`? Let's just inline this and write `defaultValue= 1L`.